ui/app/components/ui/list-item/list-item.component.js

```diff
@@ -0,0 +1,138 @@
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import Approve from '../icon/approve-icon.component'
+import Interaction from '../icon/interaction-icon.component'
+import Preloader from '../icon/preloader'
+import Send from '../icon/send-icon.component'
+
+const SendIcon = () => (
+  <Send
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const InteractionIcon = () => (
+  <Interaction
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const ApproveIcon = () => (
+  <Approve
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const FailIcon = () => (
+  <Interaction
+    className="list-item__icon"
+    size={28}
+    color="#D73A49"
+  />
+)
+
+const Item = ({
+  className,
+  status,
+  title,
+  subtitle,
+  children,
+  nativeCurrency,
+  currentCurrency,
+}) => {
+  const isApproved = status === 'approved'
+  const isUnapproved = status === 'unapproved'
+  const isPending = status === 'pending'
+  const isFailed = status === 'failed'
+
+  if (isUnapproved) {
+    subtitle = (
+      <h3 className="list-item__subheading">
+        <span className="list-item__status--unapproved">Unapproved</span> · {subtitle}
+      </h3>
+    )
+  }
+
+  if (isFailed) {
+    subtitle = (
+      <h3 className="list-item__subheading">
+        <span className="list-item__status--failed">Failed</span> · {subtitle}
+      </h3>
+    )
+  }
+
+  return (
+    <div className={className}>
+      <div className="list-item__col">
+        {isApproved ? (
+          <ApproveIcon />
+        ) : isPending ? (
```

Comment:
Nit: nested ternaries are discouraged, and are already disallowed via eslint in the upcoming version of our eslint config.

Maybe this could be converted to a set of `if ... else if`blocks that sets an icon variable